### ISSUE TITLE
MLB 11 The Show Graphical Issue

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#14046]
### Quick summary

The Crowd seems to be on the pitch behind the pitcher and the batter seems to be invisible. The issues don't appear on the ps3.

### Details

Rpcs3:
![NPUA70124_screenshot_2023_06_18_07_52_13](https://github.com/RPCS3/rpcs3/assets/71127352/e1dff41a-3e83-4707-ba5f-10d7caf032e2)

Ps3:
https://youtu.be/IThJ1ccHPs8?t=206

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11780812/RPCS3.log.gz)

### System configuration

RPCS3 v0.0.28-15213-e2bced97 Alpha | master | FW 4.88 | 5.15.106-1-MANJARO
AMD Ryzen 5 2600 | 12 Threads | 15.95 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (530.41)

**Comment by Asinin3** [comment]
Needs renderdoc or rsx capture.

**Comment by Triticum0** [comment]
Retested with v0.0.30-15914-2a1af17f. No changes

**Comment by digant73** [comment]
is that still an issue?

**Comment by kd-11** [comment]
This bug is hilarious :joy:

**Comment by Triticum0** [comment]
Yes it still an issue

**Comment by michael-rutledge** [comment]
**TL;DR: the provided logs are packed full of warnings/errors, and I tried my best to relay relevant logs here.**

---

I looked at the original log file and cross-referenced with the [MLB 16 tracker](https://github.com/RPCS3/rpcs3/issues/14731) log file, and found the following consistencies:

### `sys_fs_open` failures on what appear to be game-specific files

The following files all fail on `sys_fs_open` with `CELL_ENOENT`, or ["file does not exist"](https://github.com/RPCS3/rpcs3/blob/8f066541a3c3d8f0d830872f42419be3a4cbf1c3/rpcs3/Emu/Cell/ErrorCodes.h#L109):
* `/dev_hdd0/game/NPUA70124/USRDIR/PATCH_GD.WAD`
* `/dev_hdd0/game/NPUA70124/USRDIR/PATCH_GD.WAD.HDR`
* `/dev_hdd0/game/NPUA70124/USRDIR/PXP_SAVE.BIN`
* `/dev_hdd0/game/NPUA70124/USRDIR/PA_SAVE.BIN`
* `/dev_hdd0/game/NPUA70124/USRDIR/playlist.dat`

### Lots of "swizzled" data warnings

Example: `RSX: [Performance warning] CPU readback of swizzled data`

"Swizzled" data has been relevant to graphical issues before (https://github.com/RPCS3/rpcs3/issues/8150), so perhaps it's relevant here?

### Lots of cache misses

Example: `Cache miss at address x. This is gonna hurt...`

But according to [this other comment](https://github.com/RPCS3/rpcs3/issues/8773#issuecomment-895414155) on an unrelated issue though, maybe not relevant?

### Lots of `cellGemGetInfo` warnings
Example: `cellGem: cellGemGetInfo(...)`

These are all logged at the warning level and typically come in a chain of three logs that look like this:

```
·W ... cellGem: cellGemGetInfo(...)
·! ... cellCamera: cellCameraReadEx(...)
·U ... cellCamera TODO: cellCameraGetType(...)
```

---

P.S.: I am new to the emulator, so pardon any ignorance coming through here.

**Comment by michael-rutledge** [comment]
Did some of my own local testing to drill down into my previous comment, and I found a new log statement that triggers basically every frame when this specific crowd rendering glitch happens (i.e. does not trigger during the missing texture face glitch from the MLB 16 bug):

```
W RSX: Invalid transform register index (load=467, index=0, count=16)
W RSX: Invalid transform register index (load=467, index=4, count=12)
```

This log statement is defined in [`rpcs3/Emu/RSX/NV47/HW/nv4097.cpp`](https://github.com/RPCS3/rpcs3/blob/7cfe96a1d121e12fbdfe42befe9af5bbd561be99/rpcs3/Emu/RSX/NV47/HW/nv4097.cpp#L77), and I did see it pop up quite frequently in the original log from this report. It is logged regardless of GPU renderer.

**Comment by 127001Dev** [comment]
Hopefully this gets fixed eventually! I have found it to be less drastic on opengl rendering and some stadiums do better overall whereas others are basically unplayable since they have so much corruption.
